Editorial: authorization details in token requests/responses (#556)
openid/OpenID4VCI: openid-4-verifiable-credential-issuance-1_0.md

    credential_configuration_id%22%3A%20%22UniversityDegreeCredential%22%7D%5D
```

> The Wallet SHOULD not send in the Token Request `authorization_details` if it has been previosuly sent in the Authorization Request(#credential-authz-request)

Author: ```suggestion
The Wallet SHOULD not send the `authorization_details` parameter in the Token Request if the parameter has been previously sent in the Authorization Request(#credential-authz-request)
```
```suggestion
The Wallet SHOULD not send in the Token Request `authorization_details` if it has been previosuly sent in the Authorization Request(#credential-authz-request)
```

> The Wallet SHOULD NOT send the `authorization_details` parameter in the Token Request if the parameter has been previously sent in the Authorization Request(#credential-authz-request)

Author: @jogu I think the discussion in [RFC9396](https://www.rfc-editor.org/rfc/rfc9396.html#name-comparing-authorization-det) is relevant here. I think I should rewrite this to defer to that section and make a statement about how the token request can ask for fewer permissions than the previous request. 

Do you think an example is needed - I would say not since unlike in Rich Authz, it's an unlikely scenario

> The Wallet may send the `authorization_details` parameter in the Token Request even when the parameter has been previously sent in the [Authorization Request](#credential-authz-request) as described in Section 6.1 of [@!RFC9396]. It allows the AS to make a decision based on whether the Wallet is asking for more or less access than the previous request. With respect to `authorization_details` items using the `credential_configuration_id` introduced in this specification, it is RECOMMENDED that a AS would accept a request from the Wallet containiing a subset of `credential_configuration_id` parameters received in the original [Authorization Request](#credential-authz-request) and issue a token for the reduced set.

Reviewer: ```suggestion
The Wallet may send the `authorization_details` parameter in the Token Request even when the parameter has been previously sent in the [Authorization Request](#credential-authz-request) as described in Section 6.1 of [@!RFC9396]. It allows the AS to make a decision based on whether the Wallet is asking for more or less access than the previous request. With respect to `authorization_details` items using the `credential_configuration_id` introduced in this specification, it is RECOMMENDED that a AS would accept a request from the Wallet containing a subset of `credential_configuration_id` parameters received in the original [Authorization Request](#credential-authz-request) and issue a token for the reduced set.
```

In addition to the response parameters defined in [@!RFC6749], the Authorization Server MAY return the following parameters:

> * `authorization_details`: REQUIRED when the `authorization_details` parameter is used in either the [Authorization Request](#credential-authz-request) or [Token Request](#token-request) as defined in (#authorization-details). OPTIONAL when `scope` parameter was used to request issuance of a Credential of a certain Credential Configuration. It is a non-empty array of objects, as defined in Section 7 of [@!RFC9396]. In addition to the parameters defined in (#authorization-details), this specification defines the following parameter to be used with the authorization details type `openid_credential` in the Token Response:

Author: ```suggestion
* `authorization_details`: REQUIRED when the `authorization_details` parameter,  as defined in (#authorization-details), is used in either the [Authorization Request](#credential-authz-request) or [Token Request](#token-request). OPTIONAL when `scope` parameter was used to request issuance of a Credential of a certain Credential Configuration. It is a non-empty array of objects, as defined in Section 7 of [@!RFC9396]. In addition to the parameters defined in (#authorization-details), this specification defines the following parameter to be used with the authorization details type `openid_credential` in the Token Response:
```
```suggestion
* `authorization_details`: REQUIRED when the `authorization_details` parameter is used in either the [Authorization Request](#credential-authz-request) or [Token Request](#token-request) as defined in (#authorization-details). OPTIONAL when `scope` parameter was used to request issuance of a Credential of a certain Credential Configuration. It is a non-empty array of objects, as defined in Section 7 of [@!RFC9396]. In addition to the parameters defined in (#authorization-details), this specification defines the following parameter to be used with the authorization details type `openid_credential` in the Token Response:
```

>   * `credential_identifiers`: REQUIRED. A non-empty array of strings, each uniquely identifying a Credential Dataset that can be issued using the Access Token returned in this response. Each of these Credential Datasets corresponds to the Credential Configuration referenced in the `credential_configuration_id` parameter. The Wallet MUST use these identifiers together with an Access Token in subsequent Credential Requests. See (#identifying_credential) for the summary of the options how requested Credential(s) are identified throughout the Issuance flow.

Author: @jogu My reading of the existing spec is that this updated sentence does make sense:
1. If `authorization_details` was sent it contained a `credential_configuration_id`
2. If `scope` was sent then it's optional for the token response to contain `authorization_details`, but if it is there then `credential_configuration_id` is a REQUIRED parameter

>   * `credential_identifiers`: REQUIRED. A non-empty array of strings, each uniquely identifying a Credential Dataset that can be issued using the Access Token returned in this response. Each of these Credential Datasets corresponds to the Credential Configuration referenced in the `credential_configuration_id` parameter. The Wallet MUST use these identifiers together with an Access Token in subsequent Credential Requests. See (#identifying_credential) for the summary of the options how requested Credential(s) are identified throughout the Issuance flow.

Reviewer 2: Thanks @sloops77 - I've read it again and actually I think your change it good. Thanks for the explanation!
